crates/cli/config-test/config.toml

```diff
@@ -0,0 +1,6 @@
+rpc_url = "https://eth-mainnet.g.alchemy.com/v2/I93POQk49QE9O-NuOz7nj7sbiluW76it"
```

Comment:
this file seems redundant as we already have an example in the config folder?

Reply:
It's a test case to check that you can set a non-default path. Can remove it if you don't think it's useful to have the test case here though.

Reply:
oh I see, that makes sense. I don't really mind either way